src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -137,21 +139,29 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
     }
   }
 
+  browsePath(path: string): void {
+    this.path = path;
+  }
+
+  getDOMElement(): HTMLElement {
+    return this.elementRef.nativeElement;
+  }
+
+  getSelectedPath(): string {
+    //TODO:how do we want to want to handle caching vs message to app to open said path
+    return this.path;
+  }
+
   loadUserHomeDirectory(): void {
     this.isLoading = true;
     const observable: Observable<any> = this.ussSrv.getUserHomeFolder()
     const subscription: Subscription = observable.subscribe(
         resp => {
           if(resp && resp.home){
             this.path = resp.home.trim();
-            if (this.path.length == 0 || this.path.charAt(0) != '/') {
-              this.path = '/';
-            }
-          } else {
-            this.path = '/';
```

Comment:
This was a bad change actually. USS's home directory will always go as low as "/" and will never be "" so this was a change that we accidentally reverted but should have kept

Reply:
ok got it. So you mean this condition will not trigger anyway since the path USS's home directory will always go as low as "/" not "" ?.